src/asm/rgbasm.5

```diff
@@ -1356,7 +1347,7 @@ environment variable if that is defined as a UNIX timestamp.
 Refer to the spec at
 .Lk https://reproducible-builds.org/docs/source-date-epoch/ .
 .Sh DEFINING DATA
-.Ss Declaring variables in a RAM section
+.Ss Allocating space in RAM
```

Comment:
Maybe we should explicit that this is only static allocation, and that dynamic allocation must be managed by user code? I think it'd avoid some confusion.

Reply:
So, change the new and old mentions of "allocation" to "static allocation"?

Reply:
I think it'd be better, yeah.